javascript/javascript3/week2/Currency_Converter/index.html

```diff
@@ -0,0 +1,409 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+    <title>Currency calculator</title>
+  </head>
+  <body>
+    <main>
+      <h1>Currency Calculator</h1>
+      <div class="main-div">
+        <div class="result"></div>
+
+        <div class="up-div">
+          <input type="number" class="input-up" aria-label="from convert" />
```

Comment:
Nice with using aria label, maybe it would be better `aria-label="Amount to convert"` Good reading https://www.aditus.io/aria/aria-label/#:~:text=aria%2Dlabel%20is%20an%20attribute,content%20as%20the%20accessibility%20label.